docs/_plugins/auto_nav_generator.rb

```diff
@@ -0,0 +1,32 @@
+# Navigation Generator is a Jekyll plugin that extracts H2 headings from each page so that it can be used as part of the generation of the navigation menu.
```

Comment:
This will need to be removed.

Reply:
Lea is working on this. We will remove this later in the day.